spec/mailers/user_mailer_spec.rb

```diff
@@ -29,7 +29,7 @@
     end
 
     it 'assigns confirmation_url for email body' do
-      @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
+      @mail.body.encoded.should match("https://localhost:3000/g/#{@group.key}")
```

Comment:
I'm not sure that this test is actually testing your change, since you're just setting a default in the test environment and then seeing that it comes through.

```
ENV['FORCE_SSL'] = true
expect(@mail.body.encoded).to match /https/
```